descriptions/api.github.com/api.github.com.yaml

```diff
@@ -69369,6 +69606,7 @@ components:
       schema:
         type: string
         enum:
+        - ''
```

Comment:
Just making sure: is this intentional?

Reply:
Yup, empty string is actually a valid input for this enum 😅